xml/bp_troubleshooting_nfs.xml

```diff
@@ -13,7 +13,35 @@
    <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
- <para>
-  TBD
- </para>
+ <sect1 xml:id="bp-troubleshooting-debug-logging">
+   <title>Logging</title>
+   <para>
+     As of &productname; &productnumber;, &ganesha; reads the configuration
+     data from <filename>/etc/ganesha/ganesha.conf</filename>.
```

Comment:
```suggestion
     data from <filename>/etc/sysconfig/ganesha</filename>.
```

In SES6 this setting was actually from `/etc/sysconfig/ganesha`, but that no longer exists in SES7.

We need to find a replacement that is either:
- write a LOG block into a RADOS user config
- add log level to the NFS ServiceSpec?

Reply:
Why have you reverted it back to the SES 6 location?

Reply:
we might want to redo this entire paragraph. `/etc/sysconfig/ganesha` does not exist anymore and `/etc/ganesha/ganesha.conf` live in the container data dir and shouldn't really be modified (bootstrap code only).

The log level is current set in the `unit.run` file, but I think we should discourage modifying that also.

I'll work towards a PR that adds a log level (e.g. daemon params) to the service spec which can act as a replacement for the sysconfig file ...